config.py

```diff
@@ -36,6 +36,15 @@ class Config(object):
     RAS_FRONTSTAGE_API_SERVICE = '{}://{}:{}/'.format(RAS_FRONTSTAGE_API_PROTOCOL,
                                                       RAS_FRONTSTAGE_API_HOST,
                                                       RAS_FRONTSTAGE_API_PORT)
+
+    RAS_SECURE_MESSAGING_PROTOCOL = os.getenv('RAS_SECURE_MESSAGING_PROTOCOL', 'http')
+    RAS_SECURE_MESSAGING_HOST = os.getenv('RAS_SECURE_MESSAGING_HOST', 'localhost')
+    RAS_SECURE_MESSAGING_PORT = os.getenv('RAS_SECURE_MESSAGING_PORT', 5050)
```

Comment:
Fixed.  Going to be a trendsetter and just have the whole url in RAS_SECURE_MESSAGING_SERVICE instead of having to piece it together from 3 arguments.

Reply:
This needs to be `SECURE_MESSAGE` not `SECURE_MESSAGING` and if we change this we'd have to change in Jenkins as well, and I think that's part of a wider conversation so don't think it should change here